internal/service/grafana/workspace_api_key.go

```diff
@@ -0,0 +1,114 @@
+package grafana
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/managedgrafana"
+	"github.com/hashicorp/aws-sdk-go-base/v2/awsv1shim/v2/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+)
+
+func ResourceWorkspaceAPIKey() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceWorkspaceAPIKeyCreate,
+		Read:   schema.Noop,
+		Update: schema.Noop,
+		Delete: resourceWorkspaceAPIKeyDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
Since this resource does not support imports, this block can possibly be removed. A similar resource can be found here: https://github.com/hashicorp/terraform-provider-aws/blob/main/internal/service/kms/ciphertext.go

Reply:
Deleted as Requested.